PerfDataTxtExtension/Properties/launchSettings.json

```diff
@@ -1,9 +1,9 @@
-{
-  "profiles": {
-    "PerfDataExtensions": {
-      "commandName": "Executable",
-      "executablePath": "C:\\Tools\\WPT\\latest\\wpa.exe",
-      "commandLineArgs": "-addsearchdir C:\\src\\Microsoft-Performance-Tools-Linux\\PerfDataExtensions\\bin\\Debug\\netstandard2.1"
-    }
-  }
+{
+  "profiles": {
+    "PerfDataExtensions": {
+      "commandName": "Executable",
+      "executablePath": "C:\\Tools\\WPT\\latest\\wpa.exe",
+      "commandLineArgs": "-addsearchdir E:\\src\\Microsoft-Performance-Tools-Linux-Android\\PerfDataTxtExtension\\bin\\Debug\\netstandard2.1"

```

Comment:
I am not sure why implications do hard coding these paths to C:\... of E:\... have here. But it was not clear why the path was change in this PR.

Reply:
this is for local debugging. This file is not excluded by .gitignore. I would normally not include changes here as they are not relevant but it's good to have 1 local example. In this case the name of the extension did change from a previous commit so it was good to include this change